activities/communication/communications-plan.md

```diff
@@ -33,17 +33,17 @@ We have separate goals and objectives for different audiences and fora.
 
 Our audience is individuals from potential public member organizations, codebase communities, donors, staff, and board members.
 
-We believe we'll find them in fora organized by/for:
+We believe we'll find them in fora organized by and for:
 
-* public tech fora, for example Code4NL
+* public technology enthusiasts, for example Code4NL
 * public organization leadership, for example CommonGround or OneTeamGov
 * public organization middle management, for example iBestuur or Apolitical
-* think tanks, for example FSFE, Ford Foundation, New America Foundation,  Beeck Center for Social Impact and Innovation or Code for America
+* think tanks, for example the Free Software Foundation Europe, Ford Foundation, New America Foundation, Beeck Center for Social Impact and Innovation or Code for America
 * potential codebase communities and stakeholders - including commercial vendors
 
 ## Success criteria and indicators for target fora
 
-Nota bene: none of these goals contain 'hype' or 'coolness' because being perceived as trendy or zeitgeisty /won't/ help us build the trustworthy reputation or longterm relationships we need to succed.
+Note: none of these goals contain 'hype' or 'coolness'. This iis because being perceived as trendy or zeitgeisty will not help us build the trustworthy reputation or longterm relationships we need to succed.
```

Comment:
```suggestion
Note: none of these goals contain 'hype' or 'coolness'. This is because being perceived as trendy or zeitgeisty will not help us build the trustworthy reputation or longterm relationships we need to succeed.
```